pcsx2/Recording/InputRecording.cpp

```diff
@@ -308,17 +337,20 @@ bool InputRecording::Create(wxString FileName, bool fromSaveState, wxString auth
 	SetToRecordMode();
 	g_InputRecordingControls.DisableFrameAdvance();
 	inputRec::log("Started new input recording");
-	inputRec::consoleLog(fmt::format("Filename {}", std::string(FileName)));
+	inputRec::consoleLog(fmt::format("Filename {}", std::string(inputRecordingData.GetFilename())));
 	return true;
 }
 
-bool InputRecording::Play(wxString fileName)
+int InputRecording::Play()
```

Comment:
If we need to return more than 2 states, I'd rather it return an `enum class` so that is reads better on the consuming side.  Returning hard-coded `int`s requires knowing the internals of this function.  Though maybe it's an indication of this function trying to do too much on it's own

Reply:
Well luckily, it's getting reworked back into a bool thanks to changes to MainFrame::SetRecordingMenu (or RecordingMenuReset in the previous iteration).